docs/markdown/IDE-integration.md

```diff
@@ -70,6 +70,15 @@ The possible values for `section` are:
 
 To set the options, use the `meson configure` command.
 
+It is also possible to get the default buildoptions without a build directory by providing the root `meson.build` instead of a build directory to `meson introspect --buildoptions`.
```

Comment:
This should mention that this is only available since version 0.50. The line should also be rewrapped to 80 characters.